pkg/controller/gc_controller_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
double-import from L-39
and can be moved to L39

Reply:
Fixed.